OpenKh.sln

```diff
@@ -146,6 +146,9 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenKh.Research.Kh2Anim", "OpenKh.Research.Kh2Anim\OpenKh.Research.Kh2Anim.csproj", "{D4AF702B-145A-4052-B347-15166C43AB3F}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenKh.Command.TexFooter", "OpenKh.Command.TexFooter\OpenKh.Command.TexFooter.csproj", "{408B6E0A-20C1-4C55-A6BF-5342F2A16668}"
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "WinShell", "WinShell", "{FBD89BEB-3D49-440C-9A3C-1B52B5053DAD}"
```

Comment:
You are referencing a certain `WinShell` here, but I do not see it in the pull request

Reply:
That's the folder, I do not know why VS decided to make it a reference to a project.